src/account/zendesk.test.ts

```diff
@@ -48,7 +53,7 @@ describe('Zendesk', () => {
         sessionId: '1234',
         numberVerifiedCentralized: true,
         network: 'mainnet',
-      } as DeviceInfo,
+      } as SupportRequestUserProperties,
```

Comment:
nit: unrelated to the changes here, but I'm wondering if we could remove the cast. Maybe using `as const` or `sastifies SupportRequestUserProperties`